tps/CMakeLists.txt

```diff
@@ -38,3 +38,12 @@ FetchContent_Declare(concurrentqueue
     URL ${CMAKE_SOURCE_DIR}/tps/concurrentqueue-1.0.4.zip
 )
 FetchContent_MakeAvailable(concurrentqueue)
+
+FetchContent_Declare(stb
+    URL ${CMAKE_SOURCE_DIR}/tps/stb-master.zip
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+)
+FetchContent_MakeAvailable(stb)
+add_library(stb INTERFACE)
+target_include_directories(stb INTERFACE ${stb_SOURCE_DIR})
```

Comment:
This was some code I found to compile the stb library manually since it is header only and does not have a CMakeLists.txt to it. Is there anything you woud do differently here?

Reply:
Seems reasonable to me. You can look at my rt_log library to see how I did it, I suspect I did pretty much the same thing